src/components/send/SendContainer.js

```diff
@@ -50,7 +50,7 @@ export function SendContainer({ match }) {
     const [success, setSuccess] = useState(null)
     const amountAvailableToSend = new BN(balance.available).sub(new BN(parseNearAmount(WALLET_APP_MIN_AMOUNT)))
     const sufficientBalance = !new BN(parseNearAmount(amount)).isZero() && (new BN(parseNearAmount(amount)).lte(amountAvailableToSend) || useMax) && isDecimalString(amount)
-    const sendAllowed = ((requestStatus && requestStatus.success !== false) || id.length === 64) && sufficientBalance && amount && !formLoader
+    const sendAllowed = ((requestStatus && requestStatus.success !== false) || id.length === 64) && sufficientBalance && amount && !formLoader && !success
 
     onKeyDown(e => {
```

Comment:
@Patrick1904 I wonder if handling key down specifically can be giving us more problems than onsubmit. Or maybe just different kind of problems.

Reply:
It should be solid after I added on `!formLoader && !success` to `sendAllowed`. I've tested it thoroughly and haven't been able to get any weird behavior

Reply:
`onSubmit` was actually the cause of most of the issues in Safari, even though I disabled the submit button. Also, we need the `onKeyDown` since the user isn't focused on any input once the confirmation modal shows